src/main/java/com/ericsson/ei/status/StatusData.java

```diff
@@ -0,0 +1,39 @@
+/*
+   Copyright 2019 Ericsson AB.
+   For a full list of individual contributors, please see the commit history.
+
+   Licensed under the Apache License, Version 2.0 (the "License");
+   you may not use this file except in compliance with the License.
+   You may obtain a copy of the License at
+
+       http://www.apache.org/licenses/LICENSE-2.0
+
+   Unless required by applicable law or agreed to in writing, software
+   distributed under the License is distributed on an "AS IS" BASIS,
+   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+   See the License for the specific language governing permissions and
+   limitations under the License.
+*/
+package com.ericsson.ei.status;
+
+import lombok.Getter;
+import lombok.Setter;
+
+/**
+ * StatusData contains variables that will be converted to a JSON formatted string when a request is
+ * made to the /status endpoint. The name reflects the key element and the status value represent the
+ * value element. Any variable added in this class will be converted to JSON and represented in the
+ * status information.
+ *
+ */
+@Getter
+@Setter
+public class StatusData {
+
+    private Status eiffelIntelligenceStatus = Status.NOT_SET;
```

Comment:
Connected to my previous comment, is this one really needed?

Reply:
Yes.